odpf/optimus/core/v1beta1/job_spec.proto

```diff
@@ -41,6 +41,14 @@ service JobSpecificationService {
     };
   }
 
+  // AddJobSpecification registers new jobs for a namespace which belongs to the given project
+  rpc AddJobSpecifications(AddJobSpecificationsRequest) returns (AddJobSpecificationsResponse) {
+    option (google.api.http) = {
+      post: "/v1beta1/project/{project_name}/namespace/{namespace_name}/add/jobs"
```

Comment:
HI @Mryashbhardwaj , is this endpoint already discussed? I believe it's better to avoid verbe in the endpoint.

Reply:
+1

Reply:
Removed the verb from the endpoint